BranchSDK/Windows/BranchIO-Static/BranchIO-Static.vcxproj

```diff
@@ -187,7 +187,7 @@
     <ClCompile>
       <WarningLevel>Level3</WarningLevel>
       <SDLCheck>true</SDLCheck>
-      <PreprocessorDefinitions>DEBUG;_DEBUG;_LIB;%(PreprocessorDefinitions)</PreprocessorDefinitions>
+      <PreprocessorDefinitions>WIN32;DEBUG;_DEBUG;_LIB;%(PreprocessorDefinitions)</PreprocessorDefinitions>
```

Comment:
What does adding WIN32 actually to ?  My understanding based on this article (https://stackoverflow.com/questions/17380340/win32-preprocessor-definition-in-64bit-windows-platform) is that it doesn't do much.  Unless we have WIN32 specific code we now want to pull in ?

Reply:
This is basically just because of an old habit of mine that the SDK uses `#ifdef WIN32` in a lot of places to introduce Windows-specific code (not intended for Unix). I guess there's a better macro to use than that. I know Poco defines something, but I suspect Windows does as well. This is only really an issue until we get rid of the Linux CI.

Reply:
Ok, got it.